app/src/main/AndroidManifest.xml

```diff
@@ -146,17 +146,12 @@
             android:windowSoftInputMode="stateAlwaysHidden" />
 
         <activity
-            android:name="com.example.musicdao.MusicService"
+            android:name="com.example.musicdao.MusicActivity"
```

Comment:
could you replace this `com.example`? Looks really weird. Shouldn't have been merged like this in the first place.